vnext/ReactUWP/Utils/Helpers.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <stdint.h>
+#include <winrt/Windows.UI.Xaml.h>
+#include <IReactInstance.h>
+#include <folly/dynamic.h>
+
+namespace winrt {
+  using namespace Windows::UI::Xaml;
+}
+
+namespace react { namespace uwp {
+
+  using namespace std;
+
+  struct ReactId {
+    int64_t tag{ 0 };
+    bool isValid{ false };
+  };
+
+  template<typename T>
+  inline typename T asEnum(folly::dynamic const& obj)
+  {
+    return (T)(obj.asInt());
+  }
+
+  ReactId getReactId(IReactInstance *instance, winrt::FrameworkElement const& fe);
```

Comment:
>getReactId [](start = 10, length = 10)

Looks like this returns the view id? Should it be getViewId() then? Seems like ReactId could refer to a number of things.

Reply:
Done